champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java

```diff
@@ -29,8 +29,19 @@ public void onClick(Player player, Gamer gamer, ClickType clickType) {
 
         player.getInventory().addItem(new ItemStack(Material.IRON_SWORD));
         player.getInventory().addItem(new ItemStack(Material.IRON_AXE));
-        player.getInventory().addItem(new ItemStack(Material.BOW));
-        player.getInventory().addItem(new ItemStack(Material.ARROW, 64));
+        if((role == Role.ASSASSIN) || (role == Role.RANGER)) {
+            player.getInventory().addItem(new ItemStack(Material.BOW));
+
+            int numArrows;
+
+            if (role == Role.RANGER) {
+                numArrows = 64;
+            }
+            else {
+                numArrows = 16;
```

Comment:
maybe 32 arrows?

numArrows = (role == Role.RANGER ? 64 : 32);

Reply:
/shrug on balancing

Reply:
The command is really only used for testing at this stage

Reply:
fair enough